task.sh

```diff
@@ -1 +1,42 @@
 #! /bin/bash
+
+BACKUP_PATH="ShopDB_backup.sql"
+DATA_BACKUP_PATH="ShopDB_data_backup.sql"
+
+if [ -z "$DB_USER" ] || [ -z "$DB_PASSWORD" ]; then
```

Comment:
Consider adding a check to ensure that the `mysqldump` and `mysql` commands are available on the system before proceeding. This can prevent errors if the tools are not installed.